libs/blocks/src/lib/fast-payment/standard/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { FluidContainer, Heading, Text } from '@deriv/quill-design';
+import {
+  CardsContainer,
+  CardsContainerProps,
+  CustomLink,
+} from '@deriv-com/components';
+import { StandaloneChevronRightRegularIcon } from '@deriv/quill-icons';
+import { LinkProps } from '@deriv-com/utils';
+
+/* eslint-disable-next-line */
+export interface FastPaymentProps {
+  title: string;
+  description?: string;
+  link?: LinkProps;
+  disclaimer?: string;
+  content: Omit<CardsContainerProps<'ContentBottom'>, 'variant'>;
+}
+
+const FastPayment: React.FC<FastPaymentProps> = ({
+  title,
+  description,
+  link,
+  disclaimer,
+  content: { cards = [], cols = 'three', dense },
+}) => {
+  return (
+    <FluidContainer className="bg-background-primary-container py-general-3xl">
+      <div className="flex flex-col items-center">
+        <Heading.H2 className="mb-1200">{title}</Heading.H2>
```

Comment:
can use `semantic-tokens`?

Reply:
I was thinking of using that but if you check the figma it is using a fixed margin in all viewports. using `semantic values` will dynamically change the spacing which we don't want to happen in this case.

Reply:
as checked the `quill-tokens`, spacing example `gap` doesn't change dynamically in different screen sizes when using `semantic`.